.github/workflows/stable-spec-tests.yml

```diff
@@ -8,7 +8,7 @@ on:
   workflow_dispatch:
 
 env:
-  FIXTURES_TAG: "verkle@v0.0.8"
+  FIXTURES_TAG: "verkle@v0.0.9-alpha-1"
```

Comment:
Created this alpha release with fixtures re-filled with the fix.
CI is green now.